.github/workflows/scraper-ci.yml

```diff
@@ -17,17 +17,19 @@ env:
   CENSUS_API_KEY: ${{ secrets.CENSUS_API_KEY }}
```

Comment:
Update CI/CD workflow to use pipenv instead of installing the old `requirements.txt` directly